posts/2019-12-03-survey-launch.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Launching the 2019 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the fourth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2019 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future.
+
+Completing this survey should take about 10–15 minutes and is anonymous unless you choose to give us your contact information. We will be accepting submissions until December 16th, and we will write up our findings a month or so afterwards to [blog.rust-lang.org]. You can also check out see [last year’s results][2018 survey].
+
+- [English][survey]
+- [Simplified Chinese]
```

Comment:
I suggest adding translated text here, like `简体中文` or `Simplified Chinese (简体中文)`, that would probably make it easier for people to find their languages. Similarly below. WDYT?

Reply:
I think if we were going to do that, we'd want to do it for all of them. I'll add that as note in the feedback document so that we make sure to do it next year.